keystrokes/src/components/Character/CharacterDetails/CharacterSpecNavigation.js

```diff
@@ -12,6 +12,10 @@ CharacterSpecNavigation.propTypes = {
 	allKeybindings: PropTypes.object,
 	setAllKeybindings: PropTypes.func,
 	createNewKeybinding: PropTypes.func,
+	keyBinding: PropTypes.number,
+	setKeybinding: PropTypes.func,
+	spec: PropTypes.number,
+	setSpec: PropTypes.func,
 };
 function CharacterSpecNavigation({
 	character,
```

Comment:
do u know if there is a better convention for passing in a ton of props, or is it just one of those things u have to do?

Reply:
It depends what the props are but in this case they look necessary